docs/en_US/NAS/Overview.md

```diff
@@ -34,11 +34,8 @@ Here are some common dependencies to run the examples. PyTorch needs to be above
 
 |Name|Brief Introduction of Algorithm|
 |---|---|
-| [SPOS](SPOS.md) | [Single Path One-Shot Neural Architecture Search with Uniform Sampling](https://arxiv.org/abs/1904.00420) constructs a simplified supernet trained with an uniform path sampling method, and applies an evolutionary algorithm to efficiently search for the best-performing architectures. |
+| [SPOS's 2nd stage](SPOS.md) | [Single Path One-Shot Neural Architecture Search with Uniform Sampling](https://arxiv.org/abs/1904.00420) constructs a simplified supernet trained with an uniform path sampling method, and applies an evolutionary algorithm to efficiently search for the best-performing architectures. _Note:: SPOS is a two-stage algorithm, whose first stage is one-shot and second stage is distributed, leveraging result of first stage as a checkpoint._|
```

Comment:
`Note::` -> `Note:`. `::` is the syntax of rst.

Reply:
@ultmaster - resolved, can you review again?